apps/src/pythonlab/PythonEditor.tsx

```diff
@@ -102,7 +102,22 @@ const PythonEditor: React.FunctionComponent = () => {
       <div>
         Output:
         {codeOutput.map((outputLine, index) => {
-          return <div key={index}>{outputLine}</div>;
+          if (outputLine.type === 'img') {
```

Comment:
non-blocking nit: Should we eventually move these types to a constants file somewhere?

Reply:
This is going to get refactored completely so I think for now I'll leave as-is

Reply:
Haha, I figured! As I typed it I knew I was just making the "obligatory seeing plaintext in a condition" comment 😂 Sounds good to me!